doc/changes/latest.rst

```diff
@@ -17,6 +17,9 @@ Fixes
 Enhancements
 ------------
 
+- New example in
+  :ref:`sphx_glr_auto_examples_07_advanced_plot_beta_series.py`
+  to demonstrate how to implement common beta series models with nilearn.
```

Comment:
```suggestion
  to demonstrate how to implement common beta series models with nilearn (:gh:`3127` by `Taylor Salo`_).
```

Reply:
Thanks for catching that @NicolasGensollen.